decko/rails/controllers/card_controller.rb

```diff
@@ -108,15 +116,16 @@ def show view=nil, status=200
     card.content = card.last_draft_content if use_draft?
 
     format = format_from_params card
+    result = render_page format, view
+    status = format.error_status || status
```

Comment:
I think you are supposed to either have a return value or alter the argument (and use it as a hidden return value).  If you do both you go to hell (and let your cocoders go through it when they stumble over your code). Dealing with error codes might be mitigating circumstances nevertheless `result, error_status = render_page ...` is probably better.

Reply:
I don't disagree, but I didn't actually change how this works.